### PR TITLE
Update tronador docs

### PR DESCRIPTION
**Pull request by ShahidHussain-srti**

### CODE REVIEW

**Comment by stakater-user** [comment]
@LilShah Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-317-59d56771`

File: content/sre/tronador/workflow.md [hunk 1]
```diff
@@ -184,8 +190,8 @@ status:
 
 ### Secrets management
 
-Secrets for the helm chart to be deployed are currently passed along from the tronador config file, to the helm release. 
-Secrets for helm chart and other required resources like image pull secret can be brought into Environment Provisioner owned namespaces using [Tronador Config](./tronador_config.html) CR. 
+Secrets for the helm chart to be deployed are currently passed along from the tronador config file, to the helm release.
+Secrets for helm chart and other required resources like image pull secret can be brought into Environment Provisioner owned namespaces using [Tronador Config](./tronador_config.html) CR.
```
Comment (MuneebAijaz) [inline content/sre/tronador/workflow.md:194]: Pls update to `Environment` ^

File: content/sre/tronador/workflow.md [hunk 2]
```diff
@@ -184,8 +190,8 @@ status:
...
 
 You can also use [Tenant Operator's](../tenant-operator/overview.html) [TemplateGroupInstance](../tenant-operator/customresources.html#_5-templategroupinstance) to pass secrets to the namespace that will be provisioned by the EnvironmentProvisioner by setting the proper label in your tronador config file. An example for this workflow is [provided here](../tenant-operator/usecases/deploying-templates.html#deploying-template-to-namespaces-via-templategroupinstances).
```
Comment (MuneebAijaz) [inline content/sre/tronador/workflow.md:196]: Pls update to `Environment` ^

File: content/sre/tronador/changelog.md [hunk 3]
```diff
@@ -1,5 +1,59 @@
 # Changelog
 
+## v0.1.4
+
+- fix: fix name of GitRepository resource referenced in HelmRelease object created by Tronador
+
+### Changes
```
Comment (MuneebAijaz) [inline content/sre/tronador/changelog.md:5]: **Changes** replicated multiple times ^

**Comment by stakater-user** [comment]
@LilShah Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-317-a466f62f`

**Review by MahnoorAsghar — APPROVED** [review]

**Comment by stakater-user** [comment]
@LilShah Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:v0.0.245`

**Pull request merged**